### DISCUSSION TITLE
Keep numbers as string just like with useFormattedDate

### DISCUSSION THREAD

**Discussion by Monokromochi** [fureweb-com/public-google-sheets-parser#42] [Ideas]
Say my value is "$2,000" in the spreadsheet but appears as "2000" in the json.
Think it would be cool if we can keep the original string as easy as with the Dates.

**Comment by fureweb-com** [comment]
Now that I think about it, I think it would be okay to add a universal option called useFormat to accomplish that purpose.

I'll try to add an option to make it so that if there is a formatted value in the spreadsheet, it will be used first!

**Comment by fureweb-com** [comment]
The 'useFormat' option is available since [v1.5.0.](https://github.com/fureweb-com/public-google-sheets-parser/releases/tag/v1.5.0)
Thank you for your conversation posting! @mnkrome

**Reply by Monokromochi** [reply]
Wow that was so fast! Thank you for this amazing library. However after using the option my values are still return as numbers? Not sure if I'm doing something wrong;;
